samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -99,9 +99,10 @@ Testing with nRF Connect for Desktop
 #. Wait until the bond is established. Verify that the UART data is received as follows::
 
       Connected xx:xx:xx:xx:xx:xx (random)
-      The discovery procedure succeeded
       Security changed: xx:xx:xx:xx:xx:xx (random) level 2
       Pairing completed: xx:xx:xx:xx:xx:xx (random), bonded: 1
+      GATT Service could not be found during the discovery
```

Comment:
Apple iOS should have GATT Service server for Service Changed characteristic support.

The 'not-found' message pops up because the ANCS_central.ncs configuration, taken from nRF5 SDK, does not have GATT Service. The ANCS client sample works despite the GATT Service is absent.